frontend/taipy/src/ScenarioViewer.tsx

```diff
@@ -104,7 +104,7 @@ interface SequencesRowProps {
     onFocus: (e: MouseEvent<HTMLElement>) => void;
     focusName: string;
     setFocusName: (name: string) => void;
-    submittable: boolean;
+    submittable: boolean | string;
```

Comment:
What would be the use of the Boolean case, beside a value like null or ""?

Reply:
Indeed it could be a string